t/26-rpc-queue-unit.t

```diff
@@ -404,7 +403,7 @@ subtest 'RPC call: handling error response from rpc server' => sub {
 
     $cg_backend->call_rpc($c, $req_storage);
     $cg_backend->_on_message(undef,
-        qq[{"message_id": "$req_id", "result": { "error": { "code": "TestError", "message_to_client": "Error message", "details":{"field":"test_field"}}}}]
+        qq[{"message_id": "$req_id", "response": {"result": { "error": { "code": "TestError", "message_to_client": "Error message", "details":{"field":"test_field"}}}}}]
```

Comment:
looks horrible. Should we rename response to result?

Reply:
IDTS, we expect `response` as a key receive from the consumer.